src/search-manager/data/api.ts

```diff
@@ -303,6 +305,26 @@ export async function fetchSearchResults({
     limit,
   });
 
+  // Fourth query is to et a mix of hits and collections, with all the filters applied.
+  queries.push({
+    indexUid: indexName,
+    q: searchKeywords,
+    filter: [
+      // top-level entries in the array are AND conditions and must all match
+      // Inner arrays are OR conditions, where only one needs to match.
+      ...typeFilters,
+      ...extraFilterFormatted,
+      ...tagsFilterFormatted,
+    ],
+    attributesToHighlight: ['display_name', 'description', 'published'],
+    highlightPreTag: HIGHLIGHT_PRE_TAG,
+    highlightPostTag: HIGHLIGHT_POST_TAG,
+    attributesToCrop: ['description', 'published'],
+    sort,
+    offset,
+    limit,
+  });
+
```

Comment:
I believe we shouldn't need the fourth query or the third query.

Instead, the tabs "All Content", "Components", and "Collections" should just affect the `extraFilter`, and there should be only one implementation of the search results / home page view that displays whatever cards are in the results, with that filter applied.

Reply:
You should be able to _remove_ a significant amount of code with this PR, not add.